src/StartMenu.ts

```diff
@@ -296,64 +299,86 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const plugin = getPlugin(EXTENSION_POINT_ID, EXTENSION_ID);
```

Comment:
why can't you just extend the plugin/extension point description accessible via `this.desc` -> see constructor arguments